<launch>
    <!-- <arg name="target_frame_id"   default="base_link" />
    <param name="target_frame_id"   value="$(arg target_frame_id)" />

    <arg name="source_frame_id"   default="odom"/>
    <param name="source_frame_id"   value="$(arg source_frame_id)" />

    <param name="output_rate"       value="30" />

    <param name="viso_roll"          value="0" />
    <param name="viso_pitch"         value="-0.0" />
    <param name="viso_yaw"           value="1.5707963" />
    <param name="gamma_world"       value="-1.5707963" /> -->

    <node name="frame_transformer_node" pkg="ty_autopilot_core" exec="frame_transformer_node"  output="screen" />
        <!-- <remap from="robot_pose_out" to="/mavros/vision_pose/pose" /> -->

</launch>